src/components/eventCard.js

```diff
@@ -51,6 +52,12 @@ function Header({type, event}) {
     </div>
 }
 
+const DiscreetParagraph = styled.p`
```

Comment:
please move this styles to css file, to remove styles component library.

Reply:
you can replace DiscreetParagraph component with a P html native tag, add a css class and move this styles